src/l500/l500-color.cpp

```diff
@@ -105,6 +105,16 @@ namespace librealsense
                 { 2.f, "60Hz" },
                 { 3.f, "Auto" }, }));
 
+
+#ifdef __ANDROID__
+        // option to enable workaround for help weak usb hosts to support L515 devices
+        // with improved performance and stability
+        if(_fw_version >= firmware_version( "1.4.0.66" ) )
+        {
+            color_ep->register_option(RS2_OPTION_ENABLE_WEAK_USB_HOST_WA, std::make_shared<librealsense::float_option>(option_range{ 0, 1, 1, 1 }));
```

Comment:
Consider using bool_option: would make this a little more readable

Reply:
So this defaults to on, regardless of processor/environment? In all Android compilations? That's the way we want it?

Reply:
used the float option in case we need to set more level of settings, currently just a single setting.

yes, we would like to turn this on by default on all android devices, because we found other platform like pixel may have similar issue, just in much less severe degree.